Editor/Utility/CinemachineSceneTools.cs

```diff
@@ -364,28 +437,29 @@ public static void NearFarClipHandle(CinemachineVirtualCameraBase vcam, Serializ
             {
                 nearClipPlane.floatValue += 
                     SliderHandleDelta(newNearClipPos, nearClipPos, camForward);
+                if (!vcamLens.Orthographic)
+                {
+                    nearClipPlane.floatValue = Mathf.Max(0.01f, nearClipPlane.floatValue);
```

Comment:
Here you're clamping to 0.01 but inspector clamps to 0.001.  Should be consistent.